experiments/heroku/README.md

```diff
@@ -0,0 +1,47 @@
+# Heroku Deployment for cell×gene
+
+## Quick Start
+
+Clicking on the following button will forward you to heroku to begin the deployment process
+
+<a href="https://heroku.com/deploy?template=https://github.com/chanzuckerberg/cellxgene/tree/heroku">
+  <img src="https://www.herokucdn.com/deploy/button.svg" alt="Deploy">
+</a>
+
+If not already logged in to heroku, there you will be prompted to login or sign up for an account.
+
+Once logged in you will be sent to the setup page. Here you can set some of the basic settings for the app:
+
+#### Default Settngs
+
+- `App name`: the unique name name for your deployment.
+  - This will also serve as the default url (e.g. https://cellxgene.herokapp.com/)
+- `App owner`: Who will own this app. Either you personally or an organiziation/team
+- `Region`: Location of server where app will be deployed (EU or US)
+
+#### Config Vars
+
+- `DATASET`: A _publicly_ accessable URL pointing to a .h5ad file to view
+  - This is defaulted to pbm3k.h5ad
+
+After filling out the settings and pressing the `Deploy app` button heroku will begin building your deployment. This process will take a few minutes, but once completed you will have a personal free hosted version of cell×gene!
+
+## What Is Heroku and Why Should I Use It?
+
+> Heroku is a cloud platform that lets companies build, deliver, monitor and scale apps — we're the fastest way to go from idea to URL, bypassing all those infrastructure headaches.
+
+A Heroku deployment of cell×gene means that the app is not running on your local machine. Instead, the app is installed, configured, and ran on the Heroku servers (read: cloud).
+
+What this enables is a quick, non-technical method of setting up a cell×gene instance. No command line knowledge needed. This also allows machines anywhere to access the instance, so sharing a visualized dataset is as simple as sharing a link.
+
+If you want to quickly deploy an instance of cellxgene and, if not using a small dataset, okay with paying for a tiered dyno, this is a viable solution.
+
+## Why Shouldn't I Use It?
```

Comment:
Thanks for explaining this! I learned some things :)
Quick note: "dyno" is confusing to a heroku newbie.

Existing text is generally good. Some minor suggested rephrasing:
>"When is Heroku maybe not a great option?"
>There are 3 main limitations to hosting with Heroku:
>1 - Hosting deployments of cellxgene with h5ad files larger than about 512MBs will require a paid container. You can find more information about these costs [here](). Note that this can become quite expensive for large datasets!

>2 - Because of cellxgene's client-server model (see above), having multiple simultaneous users requires more memory. This means that the free container size is easily overwhelmed by multiple users, even with small datasets; this can be addressed by purchasing a larger container size.

>3 - Your dataset must be publicly available to use this one-click deployment method. (can we link to a guide they can follow if they want to set it up themselves?)

Reply:
Minor edit on 2 to make us look better :) 
2 - Having multiple simultaneous users requires more memory. This means that the free container size is easily overwhelmed by multiple users, even with small datasets; this can be addressed by purchasing a larger container size.